<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapGeneration</class>
 <widget class="QDialog" name="MapGeneration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Map Generation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="16" column="0">
      <widget class="QPushButton" name="regenerateButton">
       <property name="text">
        <string>Regenerate</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="seed_label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>0:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>Unknown:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="4" colspan="2">
      <widget class="QComboBox" name="perturbDBox">
       <property name="styleSheet">
        <string notr="true">width:80px;</string>
       </property>
       <property name="currentIndex">
        <number>5</number>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>256</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>512</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1024</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QComboBox" name="perturbFrequencyBox">
       <property name="styleSheet">
        <string notr="true">width:80px;</string>
       </property>
       <property name="currentIndex">
        <number>5</number>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>256</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>512</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1024</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>Erode:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QSpinBox" name="erodeSmoothLevelBox">
       <property name="styleSheet">
        <string notr="true">width:80px;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="5">
      <widget class="QSlider" name="seedSlider">
       <property name="maximum">
        <number>25</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>Level:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>Level:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>Frequency:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="4" colspan="2">
      <widget class="QComboBox" name="erodeBox">
       <property name="styleSheet">
        <string notr="true">width:80px;</string>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>256</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>512</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" colspan="6">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Perlin noise:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="6">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Perturb:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="6">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Erode and smooth:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="6">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Seed:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>Height:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QSpinBox" name="perlinNoiseHeightBox">
       <property name="maximum">
        <number>512</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QComboBox" name="perlinNoiseLevelBox">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>256</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>512</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>Multiple:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="4" colspan="2">
      <widget class="QSpinBox" name="perlinNoiseMultipleBox">
       <property name="maximum">
        <number>256</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="16" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="16" column="1" colspan="3">
      <widget class="QCheckBox" name="regenerateBox">
       <property name="text">
        <string>Regenerate on change</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MapGeneration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MapGeneration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
